classes/module/Module.php

```diff
@@ -70,6 +70,9 @@ abstract class ModuleCore implements ModuleInterface
     /** @var string A little description of the module */
     public $description;
 
+    /** @var string An additional description to fetch for the module */
+    public $extraDescription;
```

Comment:
The naming is hard. `extra` is not clear. Could you replace it by `additionalDescription` ? Thanks